### PR TITLE
update ruling

### PR DESCRIPTION
**Pull request by stas-vilchik**

### CODE REVIEW

File: ruling/snapshots/no-identical-expressions [hunk 1]
```diff
@@ -1,5 +1,7 @@
+src/angular.js/src/ng/directive/input.js: 1530
+src/angular.js/src/ng/directive/ngModel.js: 1091
+src/brackets/src/thirdparty/lodash.js: 42
```
Comment (vilchik-elena) [inline ruling/snapshots/no-identical-expressions]: @stas-vilchik minified. can we exclude it?

File: ruling/snapshots/no-one-iteration-loop [hunk 2]
```diff
@@ -1,2 +1,7 @@
+src/react-native/Libraries/Renderer/ReactFabric-dev.js: 12235
```
Comment (vilchik-elena) [inline ruling/snapshots/no-one-iteration-loop]: @stas-vilchik these are not generated? kind of biggy

File: ruling/snapshots/no-use-of-empty-return-value [hunk 3]
```diff
@@ -1 +1,10 @@
-
+src/brackets/src/extensions/default/InlineColorEditor/unittests.js: 410
+src/brackets/src/extensions/default/JavaScriptCodeHints/thirdparty/requirejs/require.js: 15,16,16,21,29
+src/brackets/src/thirdparty/jquery-2.1.3.min.js: 2,3
+src/freeCodeCamp/common/app/Map/redux/utils.js: 8
+src/react-native/Libraries/Renderer/ReactFabric-prod.js: 723,744,819,1621,3238,3572,4858,4942,4953,5299
+src/react-native/Libraries/Renderer/ReactNativeRenderer-prod.js: 819,840,915,1809,3503,5256,5340,5351,5697
+src/react-native/local-cli/link/ios/unlinkAssets.js: 53
+src/reveal.js/lib/js/head.min.js: 6,6,6,6,6,6
```
Comment (vilchik-elena) [inline ruling/snapshots/no-use-of-empty-return-value]: @stas-vilchik minified

File: ruling/snapshots/no-use-of-empty-return-value [hunk 4]
```diff
@@ -1 +1,10 @@
...
+src/reveal.js/plugin/highlight/highlight.js: 76
+src/three.js/editor/js/libs/glslprep.min.js: 158,164,164,165,192
```
Comment (vilchik-elena) [inline ruling/snapshots/no-use-of-empty-return-value]: @stas-vilchik min

File: ruling/snapshots/no-use-of-empty-return-value [hunk 5]
```diff
@@ -1 +1,10 @@
-
+src/brackets/src/extensions/default/InlineColorEditor/unittests.js: 410
+src/brackets/src/extensions/default/JavaScriptCodeHints/thirdparty/requirejs/require.js: 15,16,16,21,29
+src/brackets/src/thirdparty/jquery-2.1.3.min.js: 2,3
```
Comment (vilchik-elena) [inline ruling/snapshots/no-use-of-empty-return-value]: @stas-vilchik this entire dir can be dropped probably

File: ruling/snapshots/prefer-while [hunk 6]
```diff
@@ -1 +1,17 @@
...
+src/create-react-app/packages/react-error-overlay/src/utils/getSourceMap.js: 84
+src/react-native/Libraries/Renderer/ReactFabric-prod.js: 616,619,621,1249,1252,1347,1366,2197,3462,3513,3724,4090,4572,4678,4681,4782,4927,4944,5046,5065
+src/react-native/Libraries/Renderer/ReactNativeRenderer-prod.js: 712,715,717,1465,1468,1563,1582,2453,3855,3931,4166,4188,4227,4323,4365,4970,5076,5079,5180,5325,5342,5444,5463
+src/reveal.js/plugin/highlight/highlight.js: 76
+src/three.js/editor/js/libs/acorn/acorn_loose.js: 212,1220
+src/three.js/editor/js/libs/acorn/acorn.js: 1985,2725,2862,2885
+src/three.js/editor/js/libs/codemirror/addon/tern.js: 343,353
+src/three.js/editor/js/libs/codemirror/codemirror.js: 1999,2900,2931,4421,4883,4993,6748,7056,7098,8700
+src/three.js/editor/js/libs/esprima.js: 2555,2582
+src/three.js/editor/js/libs/glslprep.min.js: 23,24,26,27,36,37,39,40,40,41,43,44,46,47,51,52,65,68,72,73,74,75,90,97,99,102,104,112,117,119,122,125,126,129,131,133,135,136,138,149,150,152
```
Comment (vilchik-elena) [inline ruling/snapshots/prefer-while]: @stas-vilchik min

File: ruling/snapshots/prefer-while [hunk 7]
```diff
@@ -1 +1,17 @@
-src/react/scripts/babel/__tests__/transform-prevent-infinite-loops-test.js: 30
+src/brackets/src/extensions/default/QuickView/main.js: 302
+src/brackets/src/thirdparty/jquery-2.1.3.js: 1977,1985
+src/brackets/src/thirdparty/lodash.js: 13,16,21,22,24,24,24,33,35,52,55,56,78,91
+src/create-react-app/packages/react-error-overlay/src/utils/getSourceMap.js: 84
+src/react-native/Libraries/Renderer/ReactFabric-prod.js: 616,619,621,1249,1252,1347,1366,2197,3462,3513,3724,4090,4572,4678,4681,4782,4927,4944,5046,5065
+src/react-native/Libraries/Renderer/ReactNativeRenderer-prod.js: 712,715,717,1465,1468,1563,1582,2453,3855,3931,4166,4188,4227,4323,4365,4970,5076,5079,5180,5325,5342,5444,5463
+src/reveal.js/plugin/highlight/highlight.js: 76
```
Comment (vilchik-elena) [inline ruling/snapshots/prefer-while]: @stas-vilchik minified

File: ruling/snapshots/prefer-while [hunk 8]
```diff
@@ -1 +1,17 @@
-src/react/scripts/babel/__tests__/transform-prevent-infinite-loops-test.js: 30
+src/brackets/src/extensions/default/QuickView/main.js: 302
+src/brackets/src/thirdparty/jquery-2.1.3.js: 1977,1985
```
Comment (vilchik-elena) [inline ruling/snapshots/prefer-while]: @stas-vilchik thirdparty

File: ruling/snapshots/no-one-iteration-loop (hunk 2 above)
Comment (stas-vilchik) [reply ruling/snapshots/no-one-iteration-loop]: I'm not sure

**Pull request merged**